app/grandchallenge/reader_studies/models.py

```diff
@@ -1075,6 +1075,9 @@ class ScoringFunction(models.TextChoices):
     interface = models.ForeignKey(
         ComponentInterface, on_delete=models.PROTECT, null=True, blank=True
     )
+    answer_widget = models.ForeignKey(
```

Comment:
Widgets should not be re-used between questions, so this should be a OneToOne relationship.